definitions/consented/json/CaseEvent.json

```diff
@@ -675,5 +675,89 @@
     "PostConditionState": "*",
     "SecurityClassification": "Public",
     "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "ID": "FR_awaitingInfo",
+    "Name": "Awaiting Information",
+    "Description": "Awaiting Information",
+    "DisplayOrder": 38,
+    "PreConditionState(s)": "applicationSubmitted;applicationIssued;responseReceived;orderMade;consentOrderApproved",
+    "PostConditionState": "awaitingInfo",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "ID": "FR_caseSubmissionFromAwaitingInfo",
+    "Name": "Case Submission",
+    "Description": "Case Submission",
+    "DisplayOrder": 39,
+    "PreConditionState(s)": "awaitingInfo",
+    "PostConditionState": "applicationSubmitted",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "ID": "FR_issueApplicationFromAwaitingInfo",
+    "Name": "Issue Application",
```

Comment:
Same as comment above 🙂

Reply:
Same as before, the application has at that point already been issued. This is just to move it back to the state, but without actually reissuing.